### ISSUE TITLE
Add Open Source License

### ISSUE THREAD

**Issue by carabolic** [Russell91/roman#1]
I want to port your parser, specifically [9.hs](https://github.com/Russell91/roman/blob/master/09.hs), to [attoparsec](https://hackage.haskell.org/package/attoparsec). Put since there is no license attached I don't know if I'm allowed.

Would you mind others reusing your code?

**Comment by Russell91** [comment]
Yes, I normally use an MIT license, but I guess this project was small
enough that I never bothered. You are free to use the code with no license
or reference whatsoever. If you would prefer to include a license, you can
alternatively use the standard MIT license with copyright Russell Stewart
2015 at the top.

Best,
Russell

**Comment by carabolic** [comment]
OK. So I went ahead and added an open source license (MIT) see PR #2. Thanks for your fast response.
